### DISCUSSION TITLE
[Inconsistency]: No 'remove table' function for GSLeagueTable, unlike GSStoryPage and GSGoal

### DISCUSSION THREAD

**Discussion by James103** [OpenTTD/OpenTTD#11702] [Features]
### Version of OpenTTD

Since #10001

### Expected result

There should be a function `GSLeagueTable.Remove(table_id)` which takes in the ID of a custom League Table and removes that table if it exists.

Use case: A cleanup script should also be able to remove league tables, along with goals and story pages, allowing for a clean slate when swapping or removing game scripts.

### Actual result

No such function or command exists; league tables are permanent once created.

This may have been overlooked during the review phase of #10001 before the PR was merged.

### Steps to reproduce

Try to remove a League Table using GS, only to notice there's no such function.

**Comment by 2TallTyler** [comment]
@ldpl Are you willing to fix this and #11098 before we release 14.0? (So sometime in the next few months)

**Comment by ldpl** [comment]
Maybe. I didn't initially include these functions since I didn't know the use case for them. But I guess they're so straightforward, there aren't multiple ways of doing them anyway.
